minio/Makefile

```diff
@@ -0,0 +1,8 @@
+stop:
+	docker-compose down -v
```

Comment:
there are no actual volumes attached, so `-v` is unnecessary (and in general you wouldn't expect `stop` to remove them right?)

Reply:
Leaving it in there doesn't provide any extra overhead